### PR TITLE
(fix) [O3-3202] Rename `Action` column to `Modification`

### PR DESCRIPTION
**Pull request by Jexsie**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary

Renames the `Action` column to `Modification`
## Screenshots

Screenshot 2024-05-20 at 09 32 20

## Related Issue

https://openmrs.atlassian.net/browse/O3-3202

### CODE REVIEW

**Comment by pirupius** [comment]
@Jexsie thanks for the work on this. I've made a comment on the ticket asking Wamz if we really need this column. We might get rid of this entirely.

File: src/components/orders-table/orders-data-table.component.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ const OrdersDataTable: React.FC<OrdersDataTableProps> = ({
       },
       { key: "patient", header: t("patient", "Patient") },
       { key: "test", header: t("test", "Test") },
-      { key: "action", header: t("action", "Action") },
+      { key: "action", header: t("modification", "Modification") },
```
Comment (pirupius) [inline src/components/orders-table/orders-data-table.component.tsx:104]: @Jexsie as per Wamz's reply on the ticket, let's drop this column.

**Comment by Jexsie** [comment]
thanks @pirupius for the review

**Review by pirupius — APPROVED** [review]

**Comment by ojwanganto** [comment]
How is this column helpful to the care providers? What does NEW, REVISE, DISCONTINUE mean to them? Do they even understand what this should mean? If we have to show this column then I feel we need to translate these better to avoid confusion. @pirupius @slubwama

**Comment by pirupius** [comment]
@ojwanganto that's exactly why this PR's intention changed to delete this column because it didn't add any value. The PR title just wasn't updated

**Pull request merged**